auto-motor-und-sport.de.txt

```diff
@@ -9,10 +13,13 @@ strip_id_or_class: va-article-meta
 strip_id_or_class: va-article-date
 strip: //div[contains(text(), "Die aktuelle Ausgabe")]/parent::div
 # strip h2 because that is the title of the article
-strip: //h2
+strip: //h2[1]
```

Comment:
Thanks for improving the site configs. This syntax is new to me, I learned something here.